src/hooks/api/getters/useInstanceStatus/useInstanceStatus.ts

```diff
@@ -19,6 +25,15 @@ export const useInstanceStatus = (): IUseInstanceStatusOutput => {
     return {
         instanceStatus: data,
         refetchInstanceStatus: refetch,
+        isBilling: data?.plan === InstancePlan.PRO,
+        extendTrial: async () => {
```

Comment:
We should separate the getters and API calls. We have a clear separation between read only GET operations and API mutations in our hooks. Our getters are handled by useSWR, while our API calls uses the useAPI hook for error handling and loading states. This call should be in a `useInstanceStateApi` hook following the same pattern.